TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/ChangeReasonDetailsState.cs

```diff
@@ -0,0 +1,16 @@
+namespace TeachingRecordSystem.SupportUi.Pages.RoutesToProfessionalStatus;
+
+public class ChangeReasonDetailsState
```

Comment:
Does this need to be in its own class? The other journeys add these properties to the main state class.

Reply:
I thought it might be worth having e bit of re-use